### ISSUE TITLE
mac-os-fix for update

### ISSUE THREAD

**Issue by tob-har** [GrandaddyShmax/audiocraft_plus#25]
I got the updated version fixed for mac os on my own now.

But Multiband_Diffusion is not working.
I assume following the errors, because it heavy relies on torch.
Am I right, or is there a way to use it on CPU, without torch/CUDA enabled?

**Comment by GrandaddyShmax** [comment]
Still trying to figure out how MBD works, but from what i tested so far it uses far more VRAM..

**Comment by tob-har** [comment]
yes, same impression. I tried to go through the necessary PyScrips to solve occurring errors on Mac with replacing some torch related lines with eG numpy solutions. Solved some issues, but still some way to go... keep you posted if I make progress

**Comment by jasalt** [comment]
Hit this also when trying MDB, `AssertionError: Torch not compiled with CUDA enabled`, but good to know that it wouldn't probably run anyways on 16GB M1 Mac, medium and melody models run fine.
